src/features/Builder/ActivityItems/ItemConfiguration/GroupedSelectSearchController/ItemTypeTooltip/TooltipComponents/Components/Selection.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTranslation } from 'react-i18next';
+
+import { Svg } from 'components';
+import theme from 'styles/theme';
+import { StyledBodyMedium } from 'styles/styledComponents';
+
+import {
+  StyledPresentation,
+  StyledPresentationLine,
+  StyledTooltipText,
+} from '../TooltipComponents.styles';
+import { SelectionProps, SelectionUiType } from '../TooltipComponents.types';
+
+export const Selection = ({ uiType }: SelectionProps) => {
+  const { t } = useTranslation();
+  const isSingleSelection = uiType === SelectionUiType.Single;
+
+  const optionOne = (
+    <StyledTooltipText sx={{ ml: theme.spacing(1) }}>{t('option')} 1</StyledTooltipText>
+  );
+
+  const optionTwo = (
+    <StyledTooltipText sx={{ ml: theme.spacing(1) }}>{t('option')} 2</StyledTooltipText>
+  );
+
+  const optionThree = (
+    <StyledTooltipText sx={{ ml: theme.spacing(1) }}>{t('option')} 3</StyledTooltipText>
+  );
+
+  const commonProps = {
```

Comment:
this variable can be moved outside of component scope

Reply:
done